requirements.txt

```diff
@@ -1,7 +1,29 @@
 asgiref==3.2.7
+blinker==1.4
```

Comment:
@akhil-maker  dont add unnecessary dependencies. This commit needs only python-decouple to be added